Error while checking build/O2/fullCI for a7808bed0b95506b550daf667513db0b978738d0 at 2023-12-15 11:21:
```
## sw/BUILD/O2-latest/log
c++: error: unrecognized command-line option '--rtlib=compiler-rt'
c++: error: unrecognized command-line option '--rtlib=compiler-rt'
c++: error: unrecognized command-line option '--rtlib=compiler-rt'
c++: error: unrecognized command-line option '--rtlib=compiler-rt'
/sw/SOURCES/O2/12468-slc8_x86-64/0/Detectors/TPC/calibration/SpacePoints/src/TrackInterpolation.cxx:187:49: error: no matching function for call to 'o2::globaltracking::RecoContainer::getTrack(o2::dataformats::VtxTrackIndex&) const'
ninja: build stopped: subcommand failed.
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/AliceO2/12468/a7808bed0b95506b550daf667513db0b978738d0/build_O2_fullCI/pretty.html).

--- a/Detectors/TPC/calibration/SpacePoints/src/TrackInterpolation.cxx
+++ b/Detectors/TPC/calibration/SpacePoints/src/TrackInterpolation.cxx
@@ -160,6 +160,7 @@ void TrackInterpolation::prepareInputTrackSample(const o2::globaltracking::RecoC
       int idMin = vtref.getFirstEntryOfSource(is), idMax = idMin + vtref.getEntriesOfSource(is);
       for (int i = idMin; i < idMax; i++) {
         auto vid = trackIndex[i];
+        auto vidOrig = vid; // in case only ITS-TPC tracks are configured vid might be overwritten. We need to remember it for the PID
         if (mParams->ignoreNonPVContrib && !vid.isPVContributor()) {
           continue;
         }
@@ -183,6 +184,7 @@ void TrackInterpolation::prepareInputTrackSample(const o2::globaltracking::RecoC
           continue;
         }
         mSeeds.push_back(mRecoCont->getITSTrack(gidTable[GTrackID::ITS]).getParamOut());
+        mSeeds.back().setPID(mRecoCont->getTrackParam(vidOrig).getPID());
         mGIDs.push_back(vid);
         mGIDtables.push_back(gidTable);
         mTrackTimes.push_back(pv.getTimeStamp().getTimeStamp());

